public/app/features/explore/Graph/ExploreGraph.tsx

```diff
@@ -174,7 +174,14 @@ export function ExploreGraph({
     dataLinkPostProcessor,
   };
 
-  function toggleLegend(name: string, mode: SeriesVisibilityChangeMode) {
+  function toggleLegend(name: string | undefined, mode: SeriesVisibilityChangeMode) {
+    if (!name) {
+      setFieldConfig({
+        ...fieldConfig,
+        overrides: [],
+      });
+      return;
+    }
```

Comment:
Looks like this just scrolls to the logs lines now 👍 
![Screenshot 2025-06-20 at 8 10 58 AM](https://github.com/user-attachments/assets/0bd42448-2e87-4e5c-8a96-a2add666226c)